logstash-input-http_poller.gemspec

```diff
@@ -21,8 +21,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
   s.add_runtime_dependency 'logstash-codec-plain'
   s.add_runtime_dependency "logstash-mixin-http_client", ">= 7.1.0"
-  s.add_runtime_dependency 'stud', "~> 0.0.22"
```

Comment:
`stud` dependency was not used by the plugin - there was one line which seemed like a reminder of some relic.